web/templates/mention_content_wrapper.hbs

```diff
@@ -1,3 +1,6 @@
 {{~!-- --~}}
-<span class="mention-content-wrapper">{{ mention_text }}</span>
+<span class="mention-content-wrapper">{{ mention_text }}
+</span>
+{{#if is_bot}}
+    <i class="zulip-icon zulip-icon-bot" aria-label="{{t 'Bot' }}"></i>{{/if}}
 {{~!-- --~}}
```

Comment:
Given that we have Handlebars whitespace control here, does this perturb the whitespace in a problematic way?